docker/extract-sdk.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p artifacts
+
+export SDK_CONTAINER=$(docker run -d --rm orbs:sdk sleep 60)
+
+export ANDROID_EAR=client/crypto-sdk-android/crypto-sdk/build/outputs/ear
+export ANDROID_JAR=client/crypto-sdk-android/crypto-sdk/build/outputs/jar
+
+export ARTIFACTS=($ANDROID_JAR $ANDROID_EAR)
+
+for i in $ARTIFACTS; do
+    mkdir -p artifacts/$i
+    docker cp $SDK_CONTAINER:/opt/orbs/$i artifacts/$i
+done
```

Comment:
What about other SDKs? What about `client-sdk-javascript`?

Reply:
I am not sure we even need that considering you said we are not going to export build artifacts any time soon. Let’s sync about it tomorrow whenever you have time.